src/microbe_stage/components/OrganelleContainer.cs

```diff
@@ -507,6 +518,11 @@ public static void CalculateOrganelleLayoutStatistics(this ref OrganelleContaine
 
                     container.AvailableToxinTypes.TryGetValue(toxinType, out var existing);
                     container.AvailableToxinTypes[toxinType] = existing + 1;
+
+                    if (organelle.Upgrades?.CustomUpgradeData is ToxinUpgrades toxinUpgrades)
+                    {
+                        rawToxicityValue += toxinUpgrades.Toxicity;
```

Comment:
Does this affect overall toxicity of shots? I know it's hard to make stuff to do otherwise, I just don't want to have another empty review...

Reply:
It does. I clarified in other comments and in the tooltip that toxicity is averaged over the entire microbe. As the alternative would be very difficult to implement (especially when keeping save compatibility).